navigation/MainNavigator.js

```diff
@@ -26,6 +27,9 @@ function MainNavigator() {
             case "Recipe List":
               iconName = "md-search";
               break;
+            case "Profile":
+              iconName = "md-search";
```

Comment:
Looks like the icon for profile and recipe list is the same. We should change it to look unique.

Reply:
definetely.